plugin-management-library/src/main/java/pluginmanager/Plugin.java

```diff
@@ -0,0 +1,76 @@
+package pluginmanager;
+
+import java.util.jar.JarFile;
+
+public class Plugin {
```

Comment:
copy-paste?  Why do you need two similar classes?

Reply:
Are you saying that this class is too similar to the PluginManager one?  I had created a separate class because it was useful for holding multiple aspects of a plugin.

Reply:
Oh wait....I think I see what you mean. There's repeat classes in the impl and the pluginmanager folder.....yeah, that was an accident.

Reply:
Fixed in commit a78fb0dad53696ef3bd2c22a2c206e3c7caccdd1.